src/components/game/graphics/objects/cell/UpdateCell.js

```diff
@@ -0,0 +1,45 @@
+// UpdateCell refers to the graphical state of the cell, which is
+// a level above the CoreState of the game. The graphical state of the
+// cell doesn't have any impact on the CoreState, but is effectively its
+// own control system for each cell.
+
+// This cell's t parameter always decays back down to 0, and is changed in the
+// other direction only by GameState in order to light up blocks and have
+// them fade. DrawCell, UpdateCell, and the correct props are the main parts
+// that make these kinds of graphics, and much more, possible.
+const updateEmptyCell = (cell) => {
```

Comment:
These should go into the Cell class itself; they are only concerned with changing props which will become class attributes